suites/suite_metavisitor_1_2/readme.rst

```diff
@@ -0,0 +1,12 @@
+metavisitor
+
+version 1_2
+- The metavisitor suite now comes without the ncbi_blast_plus suite, which has to be installed separately from https://toolshed.g2.bx.psu.edu
```

Comment:
Yes, let's give it a try: the idea is to not force the user to search for a particular changeset. This will oblige us to keep an eye on the workflows to see whether they are still compatible with next releases of ncbi+. But it make the suite lighter to install and clearer in terms of contributions.
thus I would say: separately from https://toolshed.g2.bx.psu.edu/view/devteam/ncbi_blast_plus/   ?
...